tests/testthat/test-theme_gtsummary_roche.R

```diff
@@ -4,3 +4,26 @@ test_that("theme_gtsummary_roche() works", {
     gtsummary::check_gtsummary_theme(theme_gtsummary_roche(set_theme = FALSE))
   )
 })
+
+test_that("theme_gtsummary_roche() adds relevant {flextable} directives", {
+  # check user_added2+ are present
+  cmds <-
+    gtsummary::with_gtsummary_theme(
```

Comment:
FYI @Melkiades I added the `gtsummary::with_gtsummary_theme()` function to explicitly run these codes with the theme set.

Reply:
Very nice! Thanks Daniel